### DISCUSSION TITLE
Ask for clarification

### DISCUSSION THREAD

**Discussion by laz666** [sinaptik-ai/pandas-ai#355] [Ideas]
This is a really useful project, thank you. 

It would be really handy for PandasAI to ask for clarification if the prompt was not clear. Similar to the functionality of Code Interpreter and GPT Engineer.

**Comment by gventuri** [comment]
@laz666 that's a very great idea, in particular if we create a more interactive version (i.e. in the CLI)! I've opened a issue for that!
